cmake/modules/SIMDExt.cmake

```diff
@@ -1,66 +1,60 @@
 # detect SIMD extentions
 #
-# ARM_NEON_FLAGS
-#
 # HAVE_ARMV8_CRC
```

Comment:
the renaming of the compilation flags shoudl carefully be done in an independent commit with detailed explanations about the precautions taken to avoid missing a #ifdef HAVE_NEON or #ifdef INTEL_SSE4_1. This is specialy important because it has an impact outside of erasure code.

To be extra conservative and avoid this burden, the new HAVE_* flags could be added instead of replacing the existing flags to reduce the scope of the code change.